src/tests/fs_tests.rs

```diff
@@ -2540,6 +2552,7 @@ pub mod fs_tests {
         assert_eq!(cage.exit_syscall(libc::EXIT_SUCCESS), libc::EXIT_SUCCESS);
         lindrustfinalize();
     }
+
```

Comment:
extra space add?

Reply:
Yes, I added a new line between the 2 test functions